### PR TITLE
react-deisgn - Roland Estropia Jr.

### PR DESCRIPTION
**Pull request by restropiajr**
image

### CODE REVIEW

File: react-design/my-app/src/RotatingBanner Components/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function Banner({ items }: { items: string[] }) {
+  return <h1 className="text-3xl m-4">Display</h1>;
+}
```
Comment (TimelikeClosure) [inline react-design/my-app/src/RotatingBanner Components/Banner.tsx]: I suggest to avoid passing / requiring props that components won't use.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@restropiajr Phase 1 looks good, looking forward to Phase 2.

File: react-design/my-app/src/RotatingBanner Components/Indicators.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <div className="flex">
+      <button className="w-8 border border-solid border-black">0</button>
+      <button className="w-8 border border-solid border-black">1</button>
+      <button className="w-8 border border-solid border-black">2</button>
+      <button className="w-8 border border-solid border-black">3</button>
+      <button className="w-8 border border-solid border-black">4</button>
+      <button className="w-8 border border-solid border-black">5</button>
+    </div>
+  );
+}
```
Comment (TimelikeClosure) [inline react-design/my-app/src/RotatingBanner Components/Indicators.tsx]: @restropiajr Please make sure the number of indicators is dynamic, based on the number in base data set.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: react-design/my-app/src/RotatingBanner Components/Indicators.tsx (hunk 2 above)
Comment (restropiajr) [reply react-design/my-app/src/RotatingBanner Components/Indicators.tsx]: @TimelikeClosure updated

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@restropiajr Phase 2 looks good, looking forward to Phase 3.

**Comment by restropiajr** [comment]
Recording 2023-06-28 at 15 42 16
@TimelikeClosure completed phase 3

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**